### PR TITLE
Update hello-world.yml

### PR DESCRIPTION
**Pull request by Chazzersize**
Added 'string' in line 64 for the code to copy into the editor

## Summary
This is the first time ever sending a pull request for me.

### CODE REVIEW

**Comment by Chazzersize** [comment]
> Hi @Chazzersize 
>
> The original code was intentional. It was meant to be an assignment, after the initial declaration. 
>
> Should we make a different change to make that more clear? 

Hi @BillWagner 
Ohhh! I see that now. 
Maybe add that specific language from above identifying the terms assignment and initial declaration
Sorry I'm very newb to all of this. Thanks for having patience with me.
Do I need to do anything next?

File: docs/csharp/tour-of-csharp/tutorials/hello-world.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ items:
     of the `aFriend` variable and its initial assignment.
```
Comment (BillWagner) [inline docs/csharp/tour-of-csharp/tutorials/hello-world.yml:62]: Let's add a sentence here:

```suggestion
    of the `aFriend` variable and its initial assignment.
    
    > [!IMPORTANT]
    > Don't delete the declaration of `aFriend`. Add the code below following the existing declaration.
```

File: docs/csharp/tour-of-csharp/tutorials/hello-world.yml [hunk 2]
```diff
@@ -62,7 +62,7 @@ items:
...
 
     ```csharp
-    aFriend = "Maira";
+    string aFriend = "Maira";
```
Comment (BillWagner) [inline docs/csharp/tour-of-csharp/tutorials/hello-world.yml:65]: And then, let's revert this change:

```suggestion
    aFriend = "Maira";
```

**Review by BillWagner — COMMENTED** [review]
Thanks for chatting with me about this change. I made a couple suggestions. If you accept those, I'll merge this.

**Review by Chazzersize — COMMENTED** [review]
That will help the next learner to avoid the mistake I made.

**Review by BillWagner — APPROVED** [review]
Thanks for submitting this, and working through the conversation @Chazzersize 

I agree, this will help readers. I've reviewed everything, and I'll :shipit: as soon as the build finishes.

You should see the changes on our live site in the next day or two, on our regular publishing cycle.
